### PR TITLE
added missing release notes.

### PR DESCRIPTION
**Pull request by gknisely**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
 - **Added capability to disable narrative production** - Added the `narrative` boolean option to allow users to disable narrative production. Locations, shape, length, and time are still returned. The narrative production is enabled by default.
 - **Added capability to mark a request with an id** - The `id` is returned with the response so a user could match to the corresponding request.
 
+## Release Date: 2016-01-18
+
+- **Data producer updates** - Data importer configuration (lua) updates to fix a bug where buses were not allowed on restricted lanes.  Ferry surface issue (Changed the default surface to be compacted.).
```
Comment (dnesbitt61) [inline release-notes.md]: Walkway?

**Comment by gknisely** [comment]
@dnesbitt61 updated

File: release-notes.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 - **Added capability to disable narrative production** - Added the `narrative` boolean option to allow users to disable narrative production. Locations, shape, length, and time are still returned. The narrative production is enabled by default.
 - **Added capability to mark a request with an id** - The `id` is returned with the response so a user could match to the corresponding request.
 
+## Release Date: 2016-01-18
+
+- **Data producer updates** - Data importer configuration (lua) updates to fix a bug where buses were not allowed on restricted lanes.  Ferry surface issue (Changed the default surface to be compacted for footways.).
```
Comment (dnesbitt61) [inline release-notes.md]: How about: "Fixed surface issue (change the default surface to be "compacted" for footways)."

**Comment by gknisely** [comment]
updated

**Comment by dnesbitt61** [comment]
:shipit:

**Pull request merged**